mitodl/ocw-hugo-themes
# team listing updated: Jennifer Carter added

#### Pre-Flight checklist


#### What are the relevant tickets?
closes https://github.com/mitodl/ocw-hugo-themes/issues/1066

#### What's this PR do?
- Adds Jennifer Carter in OCW Team under OCW Staff

#### How should this be manually tested?
- Checkout this branch
- Spin up www locally (`yarn start www`)
- Go to about page and scroll to `OCW Team`
- Verify that you are able to see Jennifer Carter with correct details/spellings and image

#### Screenshots (if appropriate)
<img width="1410" alt="image" src="https://user-images.githubusercontent.com/93309234/215691337-2e984ce4-e064-4954-9da0-fa720c3883e0.png">

<img width="283" alt="image" src="https://user-images.githubusercontent.com/93309234/215691418-2f9ae89f-e671-4dc2-9cea-12c95dba4674.png">
